File: MATEMÁTICA/1170/Main.java
import java.util.Scanner;

public class Main {

    static int elapsed(float c) {
        int days = 0;
        while (c > 1) {
            days++;
            c /= 2;
        }
        return days;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        for (int times = 0; times < n; times++) {
            float c = scanner.nextFloat();
            System.out.printf("%d dias\n", elapsed(c));
        }
        scanner.close();
    }
}

File: MATEMÁTICA/1170/main.rb
# frozen_string_literal:true

def elapsed(c)
  days = 0
  while c > 1
    days += 1
    c /= 2
  end
  days
end

n = gets.to_i
n.times do
  c = gets.to_f
  puts "#{elapsed(c)} dias"
end
